### DISCUSSION TITLE
HTTPS如何快速增加SSL证书？

### DISCUSSION THREAD

**Discussion by liuquanpu** [natfrp/wiki#62] [水深火热] [answered]
HTTPS如何快速增加SSL证书？
WorkingDirectory下面放证书文件以后怎么修改配置能适配?

**Comment by fakeboboliu** [comment, marked as answer]
https://doc.natfrp.com/#/faq/site-inaccessible?id=frpc-auto-https

此功能的所有配置 case 和预期行为在上面文档中已经列出，
此功能暂时没有手把手的"小学生坐在马桶上都能读懂"级别的文档（顺便，如果哪位读者愿意贡献一篇，我们非常欢迎），
如果实在看不懂请考虑转头为 Web 服务器配置 HTTPS，不要勉强自己。

**Reply by SanaeAkahane** [reply]
> （顺便，如果哪位读者愿意贡献一篇，我们非常欢迎）

于是，[PR #83](https://github.com/natfrp/wiki/pull/83) 来咯（
